default/scripting/ship_parts/Bombard/SP_GRV.focs.txt

```diff
@@ -39,7 +39,7 @@ Part
             ]
 
         EffectsGroup    // monsters use graviton pulse on player planets
-            scope = NumberOf number = 1 condition = And [
+            scope = MaximumNumberOf number = 1 sortkey = LocalCandidate.ID condition = And [
```

Comment:
?

Reply:
jeez, thanks for catching this. kicked that unrelated change out and rebased